pages/linux/expect.md

```diff
@@ -0,0 +1,16 @@
+# expect
+
+> Programmed dialogue with interactive programs.
+> More intermation <https://linux.die.net/man/1/expect>.
+
+- Execute expect scripts from file:
+
+`expect {{file}}`
+
+- Execute expect commands from text:
```

Comment:
Perhaps this will read better?
```suggestion
- Execute a specified expect script:
```

Also, we might want to wrap `{{commands}}` in quotes.